lib/msal-common/docs/authority.md

```diff
@@ -0,0 +1,113 @@
+# Authority in MSAL
+
+In OAuth 2.0 and OpenID Connect, a client application (aka *relying party*) acquires tokens from an identity provider (IdP), such as Azure AD, to prove authentication (via ID tokens) and/or authorization (via access tokens). When doing so, the client application trusts the *authority* of the IdP.
```

Comment:
Recommend not using the term _relying party_ unless you're specifically referring only OIDC. Since the definition differs between OAuth and OIDC and SAML also refers to a relying party, it's probably safer to stick with "client" and then bring in the _authorization server_.

Maybe lean on the authorization server being the _issuer_ of the tokens--it has the _authority_ to issue those security tokens to client application.

Reply:
Removed "relying party" and slightly revised the paragraph, but kept the "IdP" instead of "AS", since we frequently use the term when responding to issues and it's somewhat more commonly used than AS in general.